config/ses.toml

```
@@ -15,6 +15,9 @@ target = "devel7"
       storage_updates = "http://download.suse.de/ibs/SUSE/Updates/Storage/7/x86_64/update/"
 
   [ses.devel7]
+    # TODO: use latest when tags is available
+    rook_ceph_chart = "registry.suse.de/suse/sle-15-sp2/update/products/ses7/charts/rook-ceph:1.4.5-1.4"
```

```suggestion
    rook_ceph_chart = "registry.suse.de/suse/sle-15-sp2/update/products/ses7/charts/rook-ceph:latest"
```

Next release of the helm chart will be published using a `latest` tag

---

Updated to use latest